plutus-benchmark/flat/Report.hs

```diff
@@ -0,0 +1,78 @@
+{-# LANGUAGE DeriveGeneric     #-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Report ( Size(..)
```

Comment:
This is a bit hard to understand without some comments: what is it doing and why? I could probably figure it out, but it requires enough effort to puzzle it out myself that my 6pm brain is giving up. Write it down!

Reply:
Added some comments.